plugins/events-backend-module-bitbucket-server/README.md

```diff
@@ -0,0 +1,43 @@
+# events-backend-module-bitbucket-server
+
+Welcome to the `events-backend-module-bitbucket-server` backend plugin!
+
+This plugin is a module for the `events-backend` backend plugin
+and extends it with an `BitbucketServerEventRouter`.
+
+The event router will subscribe to the topic `bitbucketServer`
+and route the events to more concrete topics based on the value
+of the provided `x-event-key` metadata field.
+
+Examples:
+
+| x-event-key         | topic                               |
+| ------------------- | ----------------------------------- |
+| `repo:refs_changed` | `bitbucketServer.repo:refs_changed` |
+| `repo:modified`     | `bitbucketServer.repo:modified`     |
+| `pr:opened`         | `bitbucketServer.pr:opened`         |
+
+Please find all possible webhook event types at the
+[official documentation](https://confluence.atlassian.com/bitbucketserver/event-payload-938025882.html).
+
+## Installation
+
+Install the [`events-backend` plugin](../events-backend/README.md).
+
+Install this module:
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/backend @internal/plugin-events-backend-module-bitbucket-server
```

Comment:
Need to update this to be `@backstage` scoped

Reply:
Got it, I'll fix. Was left over from when I originally had it built for my own use case.